code/api/src/migrations/92-crate-products.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('products', 'crateId', {
```

Comment:
This creates the relationship that a product is assigned to a specific crate

Reply:
Thanks for clarifying that!